modules/guide/pages/xml-generation.adoc

```diff
@@ -133,8 +133,34 @@ TIP: Simply splitting the XPath string on `'/'` won't work. You can use regular
 
 NOTE: We know that extracting the `steps` from an XPath is not rocket science and that you can do it yourself; but it would have been much easier for everyone, if we had provided this information pre-processed and ready for you inside `fields.json`. This is something we intend to do in a future release of the SDK. 
 
-=== Getting the element order right
-Ideally we would like you to be able to generate the XML without ever needing to look at the XSD. 
-For this reason we now provide the `xsdSequenceOrder` in the `fields.json`.
-One possible algorithm to use for sorting notice XML will be provided in the editor demo project, the previous algorithm was fully XSD sequence based.
+=== Getting the XML element order right
+In `fields/fields.json` a property called `xsdSequenceOrder` is available for every field that is not pointing to an XML attribute and node that is not `ND-Root`.
+You can read more about it in the link:https://docs.ted.europa.eu/eforms/latest/fields/index.html[fields documentation].
```

Comment:
I will use something like xref:sdk2/efx2.adoc[*"What's new in EFX 2"*]. In that format, but with different content